pontoon/base/models.py

```diff
@@ -354,6 +358,12 @@ class Project(models.Model):
     # Whether this project has changed since the last sync.
     has_changed = models.BooleanField(default=False)
 
+    # Aggregated stats for this project.
+    total_strings = models.PositiveIntegerField(default=0)
```

Comment:
I've never been able to decide if aggregated fields should be prefixed with `aggregate_` or something. It's super-verbose, but it's also explicit.